creusot-contracts/src/logic/ops.rs

```diff
@@ -33,10 +33,9 @@ impl<T> IndexLogic<Int> for Ghost<Seq<T>> {
     type Item = T;
 
     #[logic]
-    #[trusted]
-    #[open(self)]
-    #[creusot::builtins = "seq.Seq.get"]
-    fn index_logic(self, _: Int) -> Self::Item {
-        absurd
+    #[open]
+    #[why3::attr = "inline:trivial"]
+    fn index_logic(self, ix: Int) -> Self::Item {
+        pearlite! { (*self)[ix] }
```

Comment:
this makes some poofs harder, there is a mean increase of 7.2% in proof time

Reply:
That's not surprising unfortunately. Is this cuz he required for the PR? If not let's undo it for now.

Reply:
Previously, we translated this just to `Seq.get` which no longer works as ghost is a distinct type now.

Reply:
Are you sure that *this* is the change making things slower? How can you distinguish this from having to unpack the ghost systematically?